data/faq.yaml

```
@@ -16,37 +16,40 @@
-
+      <p>Le service est réalisé à la fabrique des géocommuns, l'incubateur de l'IGN, appliquant la méthode du programme <a href="https://beta.gouv.fr/" target="_blank">beta.gouv</a> de la Direction Interministérielle du Numérique (DINUM).</p>
+      <p>Le RNB c'est également :</p>
+      <ul>
+      <li>une <a href="https://ouverture.data.gouv.fr/donnees_de_forte_valeur?theme=G%C3%A9ospatiales&category=B%C3%A2timents&ref=referentiel-national-du-batiment.ghost.io" target="_blank">Données de forte valeur</a> définies par la <a href="https://www.data.gouv.fr/fr/posts/lessentiel-sur-les-donnees-de-forte-valeur/#:~:text=Dans" target="_blank">Directive européenne Open Data</a>. Cette Directive définit les données de forte valeur comme les « documents détenus par un organisme du secteur public, dont la réutilisation est associée à des bénéfices importants pour la société, l’environnement et l’économie ».</li>
+      <li>un <a href="https://code.gouv.fr/fr/communs/" target="_blank">commun numérique</a> que l’Etat encourage à utiliser</li>
+      <li>un chantier prioritaire de la Feuille de route Numérique et Données pour la <a href="https://www.info.gouv.fr/grand-dossier/france-nation-verte/a-proposfeuille-de-route-numerique-donnees" target="_blank">Planification Ecologique</a>, pour consolider les analyses à la maille du logement et du bâtiment</li>
+      </ul>
   - question: À qui s’adresse le Référentiel National des Bâtiments ?
     key: public
     answer: >
       <p>Le RNB est une solution destinée aux :</p>
       <ul>
-      <li>Utilisateurs de données bâtimentaires : les <b>communes</b>, les <b>métropoles</b>, les <b>fournisseurs d’énergie</b>, les services départementaux d'incendie et de secours (<b>SDIS</b>), les acteurs de l’économie numérique produisant des analyses, etc.</li>
-      <li>Producteurs de données bâtimentaires : l’ensemble des <b>administrations publiques</b> produisant des données bâtimentaires à la maille du cadastre, des parcelles ou des locaux, ainsi que celles agrégeant des données métier (historique de construction, performance énergétique, etc.)</li>
+      <li><b>Utilisateurs de données bâtimentaires</b> : les <b>communes</b>, les <b>métropoles</b>, les <b>fournisseurs d’énergie</b>, les services départementaux d'incendie et de secours (<b>SDIS</b>), les acteurs de l’économie numérique produisant des analyses, etc.</li>
+      <li><b>Producteurs de données bâtimentaires</b> : l’ensemble des <b>administrations publiques</b> produisant des données bâtimentaires à la maille du cadastre, des parcelles ou des locaux, ainsi que celles agrégeant des données métier (historique de construction, performance énergétique, etc.)</li>
       </ul>
-  - question: Qu’est ce qu’un Géocommun ?
+       - question: Qu’est ce qu’un bâtiment au sens du Référentiel National des Bâtiments (RNB) ?
```

Trop d'espaces, ça doit être aligné avec les autres questions
```suggestion
  - question: Qu’est ce qu’un bâtiment au sens du Référentiel National des Bâtiments (RNB) ?
```